packages/EasyDoctrine/src/Bridge/Symfony/Resources/config/services.php

```diff
@@ -14,5 +17,11 @@
         ->autowire()
         ->autoconfigure();
 
+    $services->set(ObjectCopierFactoryInterface::class, ObjectCopierFactory::class);
```

Comment:
I don't think we need an interface for this factory, as it is needed only for DI purposes. An interface for the copier itself - yes. But the factory is a Symfony way of registering the copier.

And, for the same reason, we should probably move the factory to a specific directory in `DependencyInjection`, as it is not related to the implementation itself.

@natepage wdyt?

Reply:
Yes I agree, no need for interface and regarding where to put the factory it's up to you guys

Reply:
@alexndlm let's put the factory to `EonX\EasyDoctrine\Bridge\Symfony\DependencyInjection` then and remove the interface. Probably, we don't even need to register it as a service in this line of code, as it is used as a service factory below.

Reply:
Refactored.